mmcv/runner/hooks/momentum_updater.py

```diff
@@ -404,11 +429,13 @@ class OneCycleMomentumUpdaterHook(MomentumUpdaterHook):
     """
 
     def __init__(self,
-                 base_momentum=0.85,
-                 max_momentum=0.95,
-                 pct_start=0.3,
-                 anneal_strategy='cos',
-                 three_phase=False,
+                 base_momentum: Union[float, List[float], Dict[str,
+                                                               float]] = 0.85,
+                 max_momentum: Union[float, List[float], Dict[str,
+                                                              float]] = 0.95,
+                 pct_start: float = 0.3,
+                 anneal_strategy: str = 'cos',
+                 three_phase: bool = False,
```

Comment:
```python
def __init__(
    self,
    base_momentum: Union[float, List[float], Dict[str, float]] = 0.85,
    max_momentum: Union[float, List[float], Dict[str, float]] = 0.95,
    pct_start: float = 0.3,
    anneal_strategy: str = 'cos',
    three_phase: bool = False,
    **kwargs,
)
```

Reply:
this one is actually from yapf, so I can't change it, if I do it will be formatted back like that

Reply:
Yes, but it is not enough readable when type hints were broken into two lines.

Reply:
We can use `# yapf: disable` to avoid formatting automatically.

Reply:
I think this would be better to make the type hint less verbose, i.e. `Union[float, list, dict]`, to make it shorter rather than use the workaround, what do you think?

Reply:
I have tried it and it passed